loops.js

```diff
@@ -19,6 +20,10 @@
 // klaar!
  // ==========================================
 
+for (let i= 0; i <5; i++) {
+    console.log("loop...");
+    if (i===4) {console.log("klaar!")};
+}
```

Comment:
Als je deze code nu uitvoert zal je zien dat hij "loop..." vijf keer print, en daarna "klaar!" (ipv vier keer "loop...")
De if statement is goed, maar bedenk wat hij "anders" zou moeten printen als hij niet aan de if voldoet 😉